draft-packages/select/KaizenDraft/Select/Select.module.scss

```diff
@@ -317,25 +316,15 @@ $focus-border-color: $color-blue-500;
       }
     }
 
-    .indicatorsContainer {
```

Comment:
Should the reference to this classname be removed from the component now as well?